### PR TITLE
tiguan 2021 ehybrid fingerprint

### PR DESCRIPTION
**Pull request by jonjander**
tiguan 2021 ehybrid fingerprint

### CODE REVIEW

File: selfdrive/car/volkswagen/values.py [hunk 1]
```diff
@@ -793,6 +797,7 @@ def init_make(self, CP: car.CarParams):
       b'\xf1\x872Q0907572Q \xf1\x890342',
       b'\xf1\x872Q0907572R \xf1\x890372',
       b'\xf1\x872Q0907572T \xf1\x890383',
+      b'\xf1\x872Q0907572T \xf1\x890383',
```
Comment (jonjander) [inline selfdrive/car/volkswagen/values.py:800]: Duplicated row

**Comment by jonjander** [comment]
> can you provide the dongle id that these came from? it will be in device settings on your comma three

ff0d015e712fd2a4

File: selfdrive/car/volkswagen/values.py (hunk 1 above)
Comment (sshane) [inline selfdrive/car/volkswagen/values.py:800]:
```suggestion
```

Comment (sshane) [inline selfdrive/car/volkswagen/values.py:800]:
```suggestion
```

**Pull request closed without merging**